orchestration-monitor/app/services/servicebus.js

```diff
@@ -0,0 +1,87 @@
+'use strict'
+
+const serviceBus = require('azure-sb')
+const eachOfLimit = require('async').eachOfLimit
+const config = require('../config')
+
+const connectionString = config.serviceBus.connectionString
+const topicName = config.serviceBus.topic
+const subscriptionName = config.serviceBus.subscription
+const maxAsyncCalls = config.serviceBus.maxAsyncCalls
+
+const serviceBusService = serviceBus.createServiceBusService(connectionString)
+
+/**
+ * @param {Array<{Object}>} batch
+ */
+async function sendServiceBusRequestBatch (batch) {
+  if (!serviceBusService) throw new Error('Failed to initialize service bus service.')
+
+  if (!batch || batch.length === 0) {
+    console.error('Attempted to send service bus empty request.')

```

Comment:
can we use a library like [debug](https://www.npmjs.com/package/debug) for logging here instead?

Reply:
will do this in a separate PR